app/constants/knights.py

```diff
@@ -0,0 +1,86 @@
+KNIGHTS = {
+    "lancelot": {
+        "name": "Lancelot",
+        "power": 35,
+        "hp": 100,
+        "armour": [],
+        "weapon": {
+            "name": "Metal Sword",
+            "power": 50,
```

Comment:
Remove the trailing comma after `"power": 50`. Trailing commas are unnecessary in the last element of a dictionary or list.